### DISCUSSION TITLE
can not delete organization

### DISCUSSION THREAD

**Discussion by jiahut** [community/community#33690] [Other Feature Feedback, Questions, & Ideas]
The **fuckxxxxxxxxx** organization has been flagged.
Because of that, your organization is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your organization’s status reviewed.

**Comment by Matteo-it** [comment]
Yes, you can't do nothing else. Contact the support center and ask for delete it.
I try to open this organization but i can't -> 404.

**Comment by jiahut** [comment]
how to remove this banner?
It exists every time it is opened, and then there is no way to remove it. 
Does no one care about the user experience?

f**k

**Reply by w3bmast3rdoteth** [reply]
I can't even vote +1 this b/c I have the same issue

**Comment by fadyat** [comment]
Is it possible to rename flagged organization? Try it

**Comment by jiahut** [comment]
> Is it possible to rename flagged organization? Try it
No chance

![image](https://user-images.githubusercontent.com/1067691/194756166-40bf1391-d40d-4aa6-b941-8bd757c8f83a.png)

**Comment by airtower-luna** [comment]
As the message says you have to [contact GitHub support](https://support.github.com/request), us other users can't help there.

**Comment by jiahut** [comment]
> As the message says you have to [contact GitHub support](https://support.github.com/request), us other users can't help there.

Came here through [contact GitHub support](https://support.github.com/request),  after came in, told me to [contact GitHub support](https://support.github.com/request) from this link, an infinite loop

**Reply by airtower-luna** [reply]
You need to pick the right category on that page (I assume for your problem it's the account issues one), not the green community forum button.

**Reply by w3bmast3rdoteth** [reply]
I am stuck in the same loop, this is pretty bad experience
